### ISSUE TITLE
Homepage logged off cleanup

### ISSUE THREAD

**Issue by agduncan94** [dockstore/dockstore#2487]
This is a collection of small changes to be made to the in progress home page.

**Comment by agduncan94** [comment]
* Make "Dockstore Tools and Workflows for the Sciences" more emphasised.
* Video button should stand out more
* DataStage Logo

**Comment by denis-yuen** [comment]
For both logged off and logged in, https://staging.dockstore.org/sitemap is not reachable directly

**Comment by NatalieEO** [comment]
- The register, browse, discover org, discover workflow buttons don't work
- Entering a search doesn't take you to the top of the search page
-  Dockstore CLI link not working

**Comment by garyluu** [comment]
The router module wasn't added which is why most links do not work.  Fix has just been merged

**Comment by denis-yuen** [comment]
>     * DataStage Logo
Elaborate on issue with datastage logo

**Comment by agduncan94** [comment]
I don't remember, but it must be adding a DataStage Logo to the homepage.

**Comment by denis-yuen** [comment]
Hmmm, pinging @Ldcabansay, do you recall anything about this?

**Comment by Ldcabansay** [comment]
I don't think we ever decided that the DataStage logo needs to be on the homepage, especially since we don't have our other funding sources on there. We did decide to display it on the grants page.

**Comment by Ldcabansay** [comment]
'Start Here' needs to be clicked twice for the page to move to overview. Otherwise everything else seems to be working correctly as of Dockstore - 1.7.0-beta.5 and UI - 2.4.0-beta.4
